db/seeds.rb

```diff
@@ -1,7 +1,41 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Author.destroy_all
+Author.create!([{
+  full_name: "Susan Cooper",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.